File: Zadania Szkopul (C++ i Java)/Login dla kazdego/Login_dla_kazdego.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{
	string a,b; // a - imie, b - nazwisko
	cin >> a >> b;
	string result ="";
	result+=a[0];
	for(int i = 1; i<a.length(); i++)
	{
		if(a[i]<b[0])
			result += a[i];
		else
		{
				break;
		}
	}
	
	result += b[0];
	cout << result;
}

File: Zadania Szkopul (C++ i Java)/Login dla kazdego/Login_dla_kazdego.java
import java.util.Scanner;

public class Login_dla_kazdego {
    public static void main(String[] args) {

        String a,b; // a - imie, b - nazwisko
        Scanner czytaj = new Scanner(System.in);
        a = czytaj.next();
        b = czytaj.next();
        czytaj.close();
        String result = "";
        result += a.charAt(0);
        for(int i = 1; i < a.length(); i++){
            if(a.charAt(i) < b.charAt(0)){
                result += a.charAt(i);
            }
            else{
                break;
            }
        }
        result += b.charAt(0);
        System.out.println(result);
    }
}
